.themes/default/source/arcs.html

```diff
@@ -2,20 +2,32 @@
 layout: page
 title: Arc Index
 rp_arcs: true
----
-
+--- 
+    
 <ul class="rp-list">
 {% for x in page['rps'] %}
   {% assign type = x[0] %}
   {% assign rp = x[1] %}
   {% if type == 'arc' %}
     <h3 class="arc-title"><a name="{{ rp.name | slugify }}" href="#{{ rp.name | slugify }}">{{ rp.name }}</a></h3>
+    <p class="arc-description">
+    {% for arc_rp in rp.rps %}
```

Comment:
But canons ought to be arcs again but different. The idea is an RP will only be in one universe. On top of that, it'd be kind of absurd to have a page with 100 different mini arc links, but like you could include a link to the main canons on the front page.